netmaster/master/netmaster.go

```diff
@@ -448,108 +447,125 @@ func CreateNetworks(stateDriver core.StateDriver, tenant *intent.ConfigTenant) e
 	}
 	rm := core.ResourceManager(tempRm)
 
-	err = validateNetworkConfig(tenant)
-	if err != nil {
-		log.Errorf("error validating network config. Error: %s", err)
-		return err
+	// Create network state
+	nwCfg := &drivers.OvsCfgNetworkState{}
+	nwCfg.StateDriver = stateDriver
+	if nwCfg.Read(network.Name) == nil {
```

Comment:
This is taken as is from CreateNetworks() code in existing netnaster. Here is what the original code looks like:
for _, network := range tenant.Networks {
		nwCfg := &drivers.OvsCfgNetworkState{}
		nwCfg.StateDriver = stateDriver
		if nwCfg.Read(network.Name) == nil {
			// TODO: check if parameters changed and apply an update if needed
			continue
		}

I think the plan was to create the network if it doesn't exist, but don't update it. Effectively, we don't handle network parameter updates.